src/main/java/org/triplea/client/launch/LaunchScreen.java

```diff
@@ -0,0 +1,654 @@
+package org.triplea.client.launch;
```

Comment:
> `org.triplea.client` as the root for all client code

This is the path I was heading down. Though it's a bit of a grey are where we would need to put code that is shared between lobby and client. In tinkering around I set a convention that shared data structures would be placed with the lobby code. That exact problem does not confront us right now, but it is the problem we are trying to fit a new package structure to.

`org.triplea.lobby` and `org.triplea.server` are package names I was working with. Open to change, but I would note that it has served well, I was able to easily delete entire features when simply removing packages. It's not an iron clad rule, but a nice heuristic I think for cohesion, it does mean that there are few external dependencies on any given package.

We are also some way off from having a proper: `org.triplea.server` . Currently, my understanding, the server game client is both a server game and a client game in one. This is why bot 'headless' clients were needed, needed a special version of the server+client bundle to host. 

Long story about `org.triplea.server`: An important change I'm looking at is to split this, so that game hosts will 'connect to themselves' and act as another client. This way the hosting code can be isolated and unified for the headless and headed cases. That should be a very large simplification to the system, we have a lot of constraints/problems due to the headless code being a copy the original server model code. Bigger picture, once the server code is unified and isolated, it would be the new "bots" and would hopefully be light weight. If not, with luck we'll be able to reason and work with that code well. Ultimately we could have bots that take up minimal and bounded amounts of memory. We would be well positioned to have many bots on just one server (would reduce TripleA ongoing costs by about 50%, $30/month; and solve the bot availability problem). We could also choose to go stateless with the servers, and avoid needing to even host `n` bots.